tests/roles/pull_openstack_configuration/tasks/main.yaml

```diff
@@ -29,9 +30,10 @@
   no_log: "{{ use_no_log }}"
   ansible.builtin.shell: |
     {{ shell_header }}
+    {{ oc_header }}
     {{ mariadb_copy_shell_vars_src }}
-    export PULL_OPENSTACK_CONFIGURATION_MYSQLCHECK_NOK=$(podman run -i --rm --userns=keep-id -u $UID $MARIADB_IMAGE \
-      mysqlcheck --all-databases -h $SOURCE_MARIADB_IP -u root "-p$SOURCE_DB_ROOT_PASSWORD" | grep -v OK)
+    export PULL_OPENSTACK_CONFIGURATION_MYSQLCHECK_NOK=$(oc run mariadb-client --image ${MARIADB_IMAGE} -i --rm --restart=Never -- \
+      mysqlcheck --all-databases -h "${SOURCE_MARIADB_IP}" -uroot -p"${SOURCE_DB_ROOT_PASSWORD}" | grep -v OK)
```

Comment:
you should also filter out unrelated lines, see how it fails in CI:
```
"+ PULL_OPENSTACK_CONFIGURATION_MYSQLCHECK_NOK='pod \"mariadb-client\" deleted'",
"+ echo 'pod \"mariadb-client\" deleted'"], "stdout": "pod \"mariadb-client\" deleted",
"stdout_lines": ["pod \"mariadb-client\" deleted"]}
```

Reply:
This was fixed by adding -q to suppress "oc run" output. Please see PR update